e2e/tests/api/helpers/delete.ts

```diff
@@ -0,0 +1,30 @@
+import type { AxiosInstance } from "axios";
+
+import { logger } from "../../common/constants";
+
+export async function deleteSboms(axios: AxiosInstance, sbomIds: string[]) {
+  var existingSbomIds = [];
+
+  for (const sbomId of sbomIds) {
+    try {
+      await axios.get(`/api/v2/sbom/${sbomId}`);
+      existingSbomIds.push(sbomId);
+    } catch (_error) {
+      logger.info(`SBOM with ID ${sbomId} does not exist anymore. Skipping.`);
+    }
+  }
```

Comment:
I think this is not necessary... since it is only cleaning we can call directly `axios.delete`, if the sbom does not exist, it will fail and nothing happens